adafruit_magtag/magtag.py

```diff
@@ -65,16 +72,16 @@ class MagTag(PortalBase):
     def __init__(
         self,
         *,
-        url=None,
-        headers=None,
-        json_path=None,
-        regexp_path=None,
-        default_bg=0xFFFFFF,
-        status_neopixel=None,
-        json_transform=None,
-        rotation=270,
-        debug=False,
-    ):
+        url: Optional[str] = None,
+        headers: Optional[Dict[str, str]] = None,
+        json_path: Optional[Sequence[Any]] = None,
+        regexp_path: Optional[Sequence[str]] = None,
+        default_bg: Union[str, int] = 0xFFFFFF,
+        status_neopixel: Optional[Union[microcontroller.Pin], neopixel.NeoPixel] = None,
```

Comment:
Looks like you've got a bracket in the wrong place, here's the fix:

```suggestion
        status_neopixel: Optional[Union[microcontroller.Pin, neopixel.NeoPixel]] = None,
```

Reply:
Sorry, I should slow down and doublecheck more often. I hate to make you kind folks catch so many mistakes!